tests/publiccloud/ipa.pm

```diff
@@ -47,28 +38,34 @@ sub find_ami {
     return;
 }
 
+sub save_logs {
+    my $output = script_output("find ipa_results -type f");
+    for my $file (split(/\n/, $output)) {
+        upload_logs($file);
+    }
+}
+
 sub run {
 
     select_console 'root-console';
 
-    if (get_required_var('PUBLIC_CLOUD_PROVIDER') != "EC2") {
-        die "Public cloud provider isn't supported";
-    }
+    die "Public cloud provider isn't supported" unless check_var('PUBLIC_CLOUD_PROVIDER', "EC2");
 
     # Install needed packages
-    zypper_call('in python3-ipa python3-ec2uploadimg aws-cli python-susepubliccloudinfo');
-
+    zypper_call('in python3-ipa python3-ipa-tests python3-ec2uploadimg python-susepubliccloudinfo git');
 
-    script_run("export AWS_ACCESS_KEY_ID=" . get_required_var('PUBLIC_CLOUD_KEY_ID'));
-    script_run("export AWS_SECRET_ACCESS_KEY=" . get_required_var('PUBLIC_CLOUD_KEY_SECRET'));
-    script_run("export AWS_DEFAULT_REGION='eu-central-1'");
+    # WAR install awscli from pip instead of using the package bcs#1095041
```

Comment:
Hm, so would it be possible to still use the packaged version of aws-cli and check for the bug condition and fall back to only trying the pip version as long as the bug is still present?

Reply:
I thought in creating a simple other test e.g. awscli, which simply install the pkg version and run a aws command to check if it is working.
What do you think?

Reply:
Sure, will work. I guess we/you can just focus on getting the real cloud part done – in whatever way :)